backend/monitor_historical_reader.go

```diff
@@ -18,7 +21,39 @@ func (r *MonitorHistoricalReader) ReadRawHistorical(ctx context.Context, monitor
 }
 
 func (r *MonitorHistoricalReader) ReadHourlyHistorical(ctx context.Context, monitorId string) ([]MonitorHistorical, error) {
-	panic("TODO: implement me!")
+	conn, err := r.db.Conn(ctx)
+	if err != nil {
+		return []MonitorHistorical{}, fmt.Errorf("failed to get connection: %w", err)
+	}
+	defer func() {
+		err := conn.Close()
+		if err != nil {
+			log.Warn().Stack().Err(err).Msg("failed to close connection")
+		}
+	}()
+
+	rows, err := conn.QueryContext(ctx, "SELECT timestamp, monitor_id, status, latency FROM monitor_historical_hourly_aggregate WHERE monitor_id = ?", monitorId)
```

Comment:
Do we need to add a limit?

Reply:
No we don't need limit.. Well maybe we would need to add limit of 1_000_000 returned records, but other than that, it should be fine